content/en/agent/kubernetes/prometheus.md

```diff
@@ -45,91 +45,90 @@ For an explanation of how Prometheus and OpenMetrics metrics map to Datadog metr
 
 ### Configuration
 
-Configure your OpenMetrics or Prometheus check using Autodiscovery, by applying the following `annotations` to your OpenMetrics/Prometheus **pod**:
+Configure your OpenMetrics or Prometheus check using Autodiscovery, by applying the following `annotations` to your **pod** exposing the OpenMetrics/Prometheus metrics:
```

Comment:
```suggestion
Configure your OpenMetrics or Prometheus check using Autodiscovery, by applying the following `annotations` to your **pod** and exposing the OpenMetrics/Prometheus metrics:
```

Reply:
Skipping this one as the pod/container itself is already exposing these metrics. Our Datadog Autodiscovery annotations are the configurations to go and collect those exposed metrics, not necessarily the configuration to expose those.